traffic_monitor/health/cache.go

```diff
@@ -46,6 +46,26 @@ const AvailableStr = "available"
 // available to serve traffic.
 const UnavailableStr = "unavailable"
 
+type Threshold int
+
+const (
+	NotEqual Threshold = iota
+	TooLow
+	TooHigh
+)
```

Comment:
Why not define a `Threshold` as a string?

```go
type Threshold string

const (
	NotEqual Threshold = "too low"
	TooLow = "too low"
	TooHigh = "too high"
)
```

That would make converting to a string easier later.

Reply:
that would be simpler.